### PR TITLE
chore(debt): remove toolbar inline styling

### PR DESCRIPTION
**Pull request by nategrift**
Part of https://github.com/PostHog/posthog/issues/16826

## Problem
- Toolbar inline styling was used instead of utility classes

## Changes

- replace the toolbar inline style that is eligible with utility classes

## How did you test this code?
- Tested Locally

### CODE REVIEW

**Comment by nategrift** [comment]
@thmsobrmlr Kept this PR small, as I wanted to hear your thoughts on the amount of `// eslint-disable-next-line react/forbid-dom-props` that is needed for future work on this issue. Was this the goal, or would you like me to further reduce how many in-line styles there are by slightly altering the UI to get the closest available values (within reason, some are required to stay style)? 

_Examples are like the 13px font size could be the 14px utility class and the extremely high zIndexes could be turned into a type of dynamic classes._

**Comment by ebrahim95** [comment]
hello,

I just wanted to ask if you got an error like this one while committing 

Command "lint-staged" not found.
husky - pre-commit hook exited with code 1

Thank You
Ebrahim Haji

**Comment by nategrift** [comment]
Hey Ebrahim! I did not get an error like that when committing. I would make sure that you follow the [Developing Locally](https://posthog.com/handbook/engineering/developing-locally) guide carefully when setting up, make sure your on your own fork of the project, make sure your committing to a new branch you create and make sure you have the python venv environment activated. 

If you are still having issues, please open a issue with more detail on your environment and I can help you out there. 😃 

> hello,
> 
> 
> 
> I just wanted to ask if you got an error like this one while committing 
> 
> 
> 
> Command "lint-staged" not found.
> 
> husky - pre-commit hook exited with code 1
> 
> 
> 
> Thank You
> 
> Ebrahim Haji

**Review by thmsobrmlr — COMMENTED** [review]
Hey @nategrift, thanks for your contribution! That's about the right amount of changes I expect. The z-index ones are really specific to the toolbar and necessary so that it works on any sites we might encounter. 

I've left a question / suggestion regarding the text size inline and once that's resolved I'm happy to merge this.

File: frontend/src/toolbar/button/ToolbarButton.tsx [hunk 1]
```diff
@@ -201,16 +201,14 @@ export function ToolbarButton(): JSX.Element {
                             opacity: inspectExtensionPercentage > 0.8 ? (inspectExtensionPercentage - 0.8) / 0.2 : 0,
                         }}
                         content={
-                            <div style={{ position: 'relative' }}>
+                            <div className="relative">
                                 <Magnifier style={{ height: 34, paddingTop: 2 }} engaged={inspectEnabled} />
                                 {inspectEnabled && selectedElement ? (
                                     <div
+                                        className="absolute top-2 left-2.25 text-white"
```
[the remark this answers is not in the document]
Comment (nategrift) [reply frontend/src/toolbar/button/ToolbarButton.tsx]: It seems to be used only in the `ToolbarApp`

**Review by thmsobrmlr — APPROVED** [review]
LGTM

**Pull request merged**